### ISSUE TITLE
Unable to create a team PR to nus-cs2103-AY1819S2/addressbook-level4

### ISSUE THREAD

**Issue by fterh** [nus-cs2103-AY1819S2/forum#14]
Hi! 

Under Week 4 Project instructions we're supposed to create a GH organization, set up our repo, etc. and make a PR to nus-cs2103-AY1819S2/addressbook-level4.

![image](https://user-images.githubusercontent.com/24870040/52335440-60634180-2a3d-11e9-98db-e84d73d4ef90.png)

I got stuck at this step, because I'm unable to make a PR to that repo - it doesn't exist as an available base repo to make a PR to.

![image](https://user-images.githubusercontent.com/24870040/52335487-81c42d80-2a3d-11e9-960a-830b999e5a40.png)

Am I missing something?

**Comment by case141** [comment]
hey fabian, 

why not try typing the full name of the repo 'nus-cs2103-AY1819S2/addressbook-level4' directly into the field? It should most likely show up this way.

**Comment by fterh** [comment]
It doesn't show up either. I'm not sure why, because from a quick google search there isn't a way to disable PRs in the first place, so it isn't like this could be an accidental setting.

**Comment by case141** [comment]
That's strange, I am able to find it though. Are you sure you entered the correct name? 
![image](https://user-images.githubusercontent.com/7495242/52344331-6402c280-2a55-11e9-9b31-2d42f3fa21e3.png)
It could also be that the PR you are trying to make to that repo is not valid.

**Comment by fterh** [comment]
Yup, I definitely entered the correct name. Anyway, I "solved" it by starting a PR from the organization's repo (rather than from nus-cs2103-AY1819S2/addressbook-level4) - when I did that, this repo showed up as an avaialble option in the list of base repositories.

Not sure why, because I was able to do a PR for addressbook-level1 without any issues even from nus-cs2103-AY1819S2/addressbook-level1.

Closing my issue, since it's resolved.
